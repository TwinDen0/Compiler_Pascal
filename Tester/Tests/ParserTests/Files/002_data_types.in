program CheckCompositeTypes;
var
a : integer;
begin
a := 5 + 5;
end.