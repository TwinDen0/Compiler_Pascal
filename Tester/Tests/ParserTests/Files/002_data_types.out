CheckCompositeTypes
├─── var
│    └─── integer
│         └─── a
├─── begin
│    └─── :=
│         ├─── a
│         └─── +
│              ├─── 5
│              └─── 5
└─── end


